templates/rhpam74-prod-immutable-kieserver-amq.yaml

```diff
@@ -1153,16 +1268,28 @@ objects:
             name: console-jolokia
             protocol: TCP
           - containerPort: 5672
-            name: amq-amqp
+            name: amqp
+            protocol: TCP
+          - containerPort: 5672
```

Comment:
shouldn't it be 5671?

Reply:
Nice catch, I think it should be, this port is set also in 73 amq ssl template https://github.com/jboss-container-images/jboss-amq-7-broker-openshift-image/blob/amq-broker-73-dev/templates/amq-broker-73-ssl.yaml#L230